lib/public/views/Logs/Logs.js

```diff
@@ -396,4 +403,82 @@ export default class Overview extends Observable {
         this.isPreviewActive = !this.isPreviewActive;
         this.notify();
     }
+
+    /**
+     * Add eligble columns for collapse to the array in the model
+     * @param {String} rowId The rowId being collapsed
+     * @param {String} name The name of the column to be collapsed
+     * @param {Integer} height Minimal height of the column
+     * @returns {undefined}
+     */
+    addCollapsableColumn(rowId, name, height) {
+        const existingColumn = this.collapsableColumns
+            .find((element) => element.rowId === rowId && element.name === name);
```

Comment:
Seeing this logic `element.rowId === rowId && element.name === name` multiple times, can't this be in any cleaner form.